receiver/otelarrowreceiver/config.go

```diff
@@ -54,3 +64,19 @@ func (cfg *ArrowConfig) Validate() error {
 	}
 	return nil
 }
+
+func (cfg *Config) Validate() error {
```

Comment:
Should we log a deprecation message during validation if a user is setting the deprecated values?

Reply:
Added a new logger using zap.Must(zap.NewDevelopment()) and am logging a warning

Reply:
Most components log their deprecation warnings about config usage in the `newReceiver...` method, which would be [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f6aa99fe3c1ebe1034de9f0d2f3ae4cab1d422e3/receiver/otelarrowreceiver/otelarrow.go#L55) for the `otelarrow` receiver. You can check if the config options are set, and log the message there using `set`.

The benefit here is you're using the passed in logger with configured settings rather than initializing your own.

_So sorry about this, I should have looked into it more when I made my original comment._

Reply:
+1

`settings.Logger` a.k.a. `settings.TelemetrySettings.Logger` should be used.

Reply:
Thanks y'all! Moved the log statements to use the set.Logger